code/modules/surgery/organ_manipulation.dm

```diff
@@ -57,7 +57,7 @@
 /datum/surgery/organ_manipulation/mechanic
 	name = "Hardware Manipulation"
 	requires_bodypart_type = BODYTYPE_ROBOTIC
-	surgery_flags = SURGERY_SELF_OPERABLE | SURGERY_REQUIRE_LIMB
+	surgery_flags = SURGERY_REQUIRE_LIMB // SKYRAT EDIT - Original SURGERY_SELF_OPERABLE | SURGERY_REQUIRE_LIMB
```

Comment:
Sorry to ask but-
Does this make synths not be able to self operate and give themselfs implants alone?
Just for clarification if im reading this right.